components/dropdown/Dropdown.jsx

```diff
@@ -30,6 +31,18 @@ class Dropdown extends React.Component {
     up: false
   };
 
+  componentWillUpdate (prevState, nextState) {
+    if (!prevState.active && nextState.active) {
+      events.addEventsToDocument({click: this.handleDocumentClick});
```

Comment:
you are totally right. I saw it when following how you did it in the other components, but then forgot completely.